backend/tests/clubs/test_security.py

```diff
@@ -138,7 +138,10 @@ def test_check_detail_level_permissions(self):
         """
 
         # Don't put your function here unless it never returns any private information.
-        whitelist = set()
+        whitelist = {
+            ("QuestionAnswerViewSet", "like"), 
```

Comment:
Looks like there's a trailing whitespace on this line now lol sorry the formatter is kinda strict
```
./tests/clubs/test_security.py:142:47: W291 trailing whitespace
```
generally it's a good idea to just run `black .` and `flake8 .` before you commit just so that the formatter will tell you want to change before you commit.

Reply:
Yeah I think I might have to push again also sorry

Reply:
Ahhhh never mind you caught it well done. I'll merge it in after these tests pass!